crates/sui-core/src/authority_aggregator.rs

```diff
@@ -161,6 +161,7 @@ where
         source_authority: AuthorityName,
         cert_handler: CertHandler,
     ) -> Result<(), SuiError> {
+        // TODO(panic): this panics
```

Comment:
panics as in it doesn't work today? or was this added by mistake?

Reply:
It's using `[]` instead of `get()` so it's a possible crashing bug in the validator. Probably doesn't crash today.